### DISCUSSION TITLE
Intro skipping isn't working on Jellyfin for Android TV

### DISCUSSION THREAD

**Discussion by eric72777** [jellyfin/jellyfin-androidtv#4316] [General]
The skip intro button or action isn't showing up on my Android TV. Did this happen to anyone else? Is there a setting I need to enable to have intro skipping work?

Versions:
Jellyfin for Android TV 0.18.2
Sony X90CH TV
Jellyfin server 10.10.3
Intro Skipper Plugin 1.0.0.7

Intro skipping works fine on my jellyfinmediaplayer 1.10.1 (Linux) once I enabled it in the application settings, but I couldn't find a similar setting in Jellyfin for Android TV.

**Comment by pavichokche** [comment]
I'm not sure what the default settings are, but there are now options in the 'Playback' section for Media segment actions. You'd have to pick something for intros/outros
